app/workers/retirement_worker.rb

```diff
@@ -15,8 +15,15 @@ def perform(count_id)
 
   def deactivate_workflow!(workflow)
     if workflow.finished?
-      workflow.active = false
-      workflow.save!
+      ignore_optimistic_lock { workflow.update_attribute(:active, false) }
     end
   end
+
+  private
+
+  def ignore_optimistic_lock
+    ActiveRecord::Base.lock_optimistically = false
```

Comment:
Yeah i don't know..I could test in staging. Any other thoughts to handle these? (see prod sidekiq retires).